### DISCUSSION TITLE
how to add an update path to a register generated from rggen

### DISCUSSION THREAD

**Discussion by nijie001** [rggen/rggen#217] [Q&A]
Is there a way to add another wr_en and wr_data to a register generated from rggen? We have an array of registers that we need to write from MCU and write from a controller. As an example, this register is 64bits, MCU will take two writes to update this register. The controller will take one write to update the whole 64bits register.  Can this be done by rggen?

**Comment by taichi-ishitani** [comment]
You can edit your config file and increase `bus_width` to 64.
https://github.com/rggen/rggen/wiki/Configuration-File#bus_width

Then, RgGen will generate CSR modules with 64 bits data bus.

**Reply by taichi-ishitani** [reply]
@nijie001 ,
Is your problem resolved?
